package.json

```diff
@@ -24,12 +26,15 @@
     "url": "https://github.com/cimasim89/flat-file-creator/issues"
   },
   "dependencies": {
+    "@types/lodash": "4.14.151",
```

Comment:
Sorry Kael, I don't know if among the things you've been advising me, you've explained why you put the @types/lodash on dependencies, or if it's a mistake... isn't it better to put it on Dev?